src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/NocNotificationService.java

```diff
@@ -46,17 +46,6 @@ public void sendNotificationOfChangeEmails(CallbackRequest callbackRequest, Case
             );
         }
 
-        String oldSolicitorEmail = NocNotificationHelper.getOldSolicitorEmail(callbackRequest);
-        if (isNullOrEmpty(oldSolicitorEmail)) {
-            log.warn("missing oldSolicitorEmail");
-        } else {
-            emailService.sendEmail(
-                previousRespondentSolicitorTemplateId,
-                oldSolicitorEmail,
-                NocNotificationHelper.buildPreviousRespondentSolicitorPersonalisation(caseData)
-            );
-        }
-
         String newSolicitorEmail = NocNotificationHelper.getNewSolicitorEmail(callbackRequest);
```

Comment:
In this Helper, I think we may need to update the getNewSolicitorEmail() to use the new CaseData to find the index of the Solicitor?
54: getIndexOfSolicitor(oldCaseData) -> getIndexOfSolicitor(newDetails)

Reply:
Refactored everything, thanks.